models/src/main/scala/coop/rchain/casper/protocol/CasperMessage.scala

```diff
@@ -147,10 +148,6 @@ object BlockMessage {
     )
 
   def toProto(bm: BlockMessage): BlockMessageProto = {
-    // To guarantee the same hash lists and maps must be sorted
-    // - can be defined on common place
-    implicit val byteStringOrdering: Ordering[ByteString] =
-      Ordering.by[ByteString, Iterable[Byte]](_.toByteArray)(Ordering.Iterable[Byte])
```

Comment:
This is duplicated from https://github.com/rchain/rchain/blob/885def3ae05786339ab5e83adc88730f8844386a/models/src/main/scala/coop/rchain/models/ByteStringSyntax.scala#L15-L16 .
I remove this one because it would cause conflict.